circuit_knitting/cutting/cut_finding/lo_cuts_optimizer.py

```diff
@@ -155,10 +152,10 @@ def get_results(self) -> DisjointSubcircuitsState | None:
         """Return the optimization results."""
         return self.best_result
 
-    def get_stats(self, penultimate=False) -> dict[str, NDArray[np.int_]]:
+    def get_stats(self, penultimate=False) -> dict[str, NamedTuple | None]:
```

Comment:
Is the idea that there might one day be more keys in this dict than just `CutOptimization`?

I wonder if this would be better.
```suggestion
    def get_stats(self, penultimate=False) -> dict[str, Any]:
```
but the docstring is still a little bit weird, because it talks about the "value" of the dict without referencing what the key(s) are.

Reply:
>Is the idea that there might one day be more keys in this dict than just CutOptimization?

Yes, that's right.

Reply:
In that case, I think my suggestion of `Any` makes the most sense.